_posts/2015-06-15-zumbatech-at-hackforchange.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Zumbatech takes on #hackforchange"
+description: How Zumbatech contributed to Hackforchange utilizing opensource tools.
+tags: [hackathon, civic coding, opensource, elasticsearch, kibana, nodejs]
+author: cjsaylor
+---
+
+On June 6, 2015, a team of engineers from Zumbatech decided to contribute in
+an all-day hackathon event called [Hackforchange](http://hackforchange.org/).
+This is a national effort for civic hacking that brings engineers and designers
+together to make a positive impact in our communities.
+
+## Let's not re-invent the wheel
+
+The day before the event, our team got together to figure out what we were going to work
+on that would give the most impact to our community. We chose to work on making visualizations
+of Florida vendor transactions. The first thing we noticed is that a couple of projects had
+already been underway to create restful APIs and alternate data formats for this data. We decided
+that an API that is specific to this data set wouldn't be very reusable for other data sets, *and*
+it would still take an engineer's effort to visualize the data from those APIs.
+
+We wanted to make something that is generic enough to work with any sort of data set,
+be flexible enough for other engineers to create tools and visualizations via an API,
+and be easy enough for non-engineers to construct visualizations that fit their needs.
+A daunting task, especially for it to be _mostly_ completed in a single hackathon.
+After some planning and discussion, we came up with [a solution ready for hacking](https://github.com/cjsaylor/datamnom)!
```

Comment:
Maybe just: "After some planning and discussion, we came up with a solution." with no link, since you link to it by name in the next section.